dataflow/src/main/java/org/checkerframework/dataflow/cfg/node/ObjectCreationNode.java

```diff
@@ -94,11 +153,13 @@ public boolean equals(@Nullable Object obj) {
     }
 
     @Override
+    @Pure
     public int hashCode() {
         return Objects.hash(constructor, arguments);
     }
 
     @Override
+    @Pure
     public Collection<Node> getOperands() {
         ArrayList<Node> list = new ArrayList<>(1 + arguments.size());
         list.add(constructor);
```

Comment:
Should this method also return `receiver`?

Reply:
Thanks! added in my local.